recipes/lapack/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR="lib" \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
+  ..
+
+make -j2
```

Comment:
Travis has some docs at https://docs.travis-ci.com/user/ci-environment/

Linux has 2 cpus, mac has 1.

CircleCI has 2 vCPUS: https://discuss.circleci.com/t/what-runs-on-the-node-container-by-default/1443